packages/web/config/chain-infos.ts

```diff
@@ -2570,6 +2570,13 @@ const chainInfos = (
           coinGeckoId: "pool:staevmos",
           coinImageUrl: "/tokens/stevmos.svg",
         },
+        {
+          coinDenom: "stINJ",
+          coinMinimalDenom: "stinj",
+          coinDecimals: 18,
+          coinGeckoId: "",
```

Comment:
the pool creator may use the preview generated in this PR to create the pool, but can you please specify an 'id' and define its price in 'config/prices.ts'?

Reply:
I don't think there's an stINJ pool yet, should I leave the `poolId` field in `price.ts` empty?

Reply:
Sure, we'll just set this PR to draft until there is a pool we can put here.

Reply:
Hey @riley-stride just want to bump on this. Any update on initial liquidity so we can list the token?